### DISCUSSION TITLE
Collection of known problems

### DISCUSSION THREAD

**Discussion by jritta** [peridigm/peridigm#146] [Ideas]
What do you think about the idea to collect all known issues of the code?
This would might help new users to get an easier access to the usage of the code. With known issues I mean, knowledge which we faced during our daily work, e.g. *do not use numbers at the beginning of the file name of an input mesh*.
When you agree to do so, what would be the best way to collect them?

Best,
Jonas

**Comment by johntfoster** [comment]
Under a discussion as you have started here.
